Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -233,7 +233,7 @@ void WebXROpaqueFramebuffer::endFrame()
 #if USE(MTLSHAREDEVENT_FOR_XR_FRAME_COMPLETION)
     if (m_completionEvent) {
         auto gCGL = static_cast<GraphicsContextGLCocoa*>(&gl);
-        auto completionSync = gCGL->createSyncWithSharedEvent(m_completionEvent, m_renderingFrameIndex);
+        auto completionSync = gCGL->createSyncWithSharedEvent(m_completionEvent.get(), m_renderingFrameIndex);
```

Comment:
Again, since you're abstacting something it'd be good to use the abstaction, e.g. work until you get to a point where you can use
```
gl.createEGLSync(m_completionEvent.get(),  m_renderingFrameIndex);

Reply:
I have the use of the change in a follow up patch, but I can move those changes here too.